components/SkilledImpact/elements/SkillContent.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { View, Text, TouchableOpacity, Switch } from 'react-native';
+import styles from '../../../constants/SkilledImpact/OrgSkilledImpactBody';
+import Sync from '../../../assets/jsicons/bottomnavigation/Sync';
+import ChevronBottom from '../../../assets/jsicons/miscIcons/ChevronBottom';
+import ChevronRight from '../../../assets/jsicons/miscIcons/ChevronRight';
+
+class SkillContent extends React.Component {
+  constructor(props) {
+    //TODO props edits and states
+    super(props);
+    this.state = {
+      skillExpand: true,
+      isReachable: props.isAcceptingHelp,
```

Comment:
**suggestion**: We're using three names for the same variable here, `accepting_help_requests` from the backend, `isAcceptingHelp` in the prop, and `isReachable` in the state. Let's standardize all of it to just `acceptingHelpRequests` to match the backend.